tech/languages/python/scipy.md

```diff
@@ -227,10 +229,33 @@ oo
 -exp(-x)
 ```
 
+## Jupyter Notebook
+
+Run and share live code via the browser; uses include: data cleaning and
+transformation, numerical simulation, statistical modeling, machine learning.
+
+First install Jupyter:
```

Comment:
I'd say "install Jupyter Notebook" otherwise it looks weird that you want to install something called Jupyter and you are dnf installing something called notebook.